runners/core-construction-java/src/test/java/org/apache/beam/runners/core/construction/ParDoTranslationTest.java

```diff
@@ -156,6 +157,54 @@ public void toAndFromTransformProto() throws Exception {
         equalTo(protoComponents.getPcollectionsOrThrow(mainInputId)));
   }
 
+  @Test
```

Comment:
These tests are going to be run `n` times because the test as a whole is parameterized.